cmd/juju/storage/volumelist.go

```diff
@@ -89,15 +104,3 @@ func (c *VolumeListCommand) Run(ctx *cmd.Context) (err error) {
 	}
 	return c.out.Write(ctx, output)
 }
-
-var getVolumeListAPI = (*VolumeListCommand).getVolumeListAPI
-
-// VolumeListAPI defines the API methods that the volume list command use.
-type VolumeListAPI interface {
-	Close() error
-	ListVolumes(machines []string) ([]params.VolumeDetailsResult, error)
-}
-
-func (c *VolumeListCommand) getVolumeListAPI() (VolumeListAPI, error) {
```

Comment:
ISTM that this approach (replacing getVolumeListAPI) could still work OK. I'm not quite seeing why we want api as an instance variable. Then the command logic remains the same as before and is arguably more obviously correct.